### ISSUE TITLE
No support for "Power Saver"

### ISSUE THREAD

**Issue by marcinwiacek** [tuxedocomputers/tuxedo-control-center#132]
Hi,

Ubuntu has got three power modes in Settings: "performance", "balanced power", "power saver", but Tuxedo Control Center seems to support only "performance" and "balanced power" (after switching to "power saver" from system Settings I see in log from the service, that it's unrecognized and profile will be reapplied).

Could you add power mode list to the profile settings (instead of performance checkbox) in the Control Center and allow user to select all modes like in Settings?

Thank you.

**Comment by tuxedoxt** [comment]
Hello, which Ubuntu version are you referring to?

**Comment by marcinwiacek** [comment]
Hello, 21.04

**Comment by tuxedoxt** [comment]
Right, we still don't base off of 21.04, it seems it has implemented some CPU settings of its own through these profiles. If TCC clashes with any other tools like this you can disable the CPU settings through *Settings* -> *Global profiles settings* -> *CPU Settings* (deactivated).
